server/bin/pbench-verify-backup-tarballs.py

```diff
@@ -91,8 +92,8 @@ def fs_entry_list_creation(self):
         self.missing_list = []
         self.error_list = []
         for tar in tarlist:
-            result_name = os.path.basename(tar)
-            controller = os.path.basename(os.path.dirname(tar))
+            result_name = Path(tar).name
```

Comment:
Let's consider making `tar` and `Path()` object first instead of creating it multiple times.